content/applications/finance/payment_acquirers/flutterwave.rst

```diff
@@ -0,0 +1,40 @@
+===========
+Flutterwave
+===========
+
+`Flutterwave <https://flutterwave.com/>`_ is an online payments provider established in Nigeria and
+covering several African countries and payment methods.
+
+.. _payment_acquirers/flutterwave/configure_dashboard:
+
+Configuration on Flutterwave Dashboard
+======================================
+
+#. Log into `Flutterwave Dashboard <https://dashboard.flutterwave.com/>`_ and go to
+   :menuselection:`Settings --> API`. Copy the values of the :guilabel:`Public Key` and
+   :guilabel:`Secret Key` fields and save them for later.
+#. | Go to :menuselection:`Settings --> Webhooks` and enter your Odoo database URL followed by
+     `/payment/flutterwave/webhook` in the :guilabel:`URL` text field.
+   | For example: `https://yourcompany.odoo.com/payment/flutterwave/webhook`.
+#. Fill the :guilabel:`Secret hash` with a password that you generate and save its value for later.
```

Comment:
It may be a good idea to give them more information than just `password that you generate`. How many characters, what tho include (symbols, numbers...). We could propose a link ready to generate a password like this one : https://passwordsgenerator.net/?length=32&symbols=1&numbers=1&lowercase=1&uppercase=1&similar=1&ambiguous=0&client=1&autoselect=0

Reply:
It could be really anything so I'd rather not let the user think that there is a constraint on the password format.